experiments/scripts/run_tune.py

```diff
@@ -103,6 +113,9 @@ def run(config_path: str, config_overrides: Optional[Dict] = None) -> None:
     trainer.train()
     trainer.save_model(config.trainer.output_dir + "/checkpoint-final")
 
+    # custom logging at end of training
+    wandb.log({"CPU": collect_cpu_memory(), "GPU": collect_gpu_memory()})
```

Comment:
maybe this could be ["CPU_end_run" and "GPU_end_run" ] with line 102 ["CPU_start_run" and "GPU_start_run" ] so it is more clear?